migrations/versions/db6b2ef8100f_.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
```

Comment:
Please add a Message:

v3.10: Add case-insensitive usernames in policies

Reply:
[b7bc726](https://github.com/privacyidea/privacyidea/pull/3852/commits/b7bc726f2279a85bad77918536eb77dbabbf0523)